platform/ios/app/Info.plist

```diff
@@ -27,9 +27,11 @@
 	<key>NSHumanReadableCopyright</key>
 	<string>© 2014–2017 Mapbox</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>The map will ALWAYS display the user’s location.</string>
+	<string>The map will always display your location.</string>
```

Comment:
This is a little misleading, since the map itself isn’t displayed when another application is in front, yet Location Services would still be active in that case. Ideally we’d be able to incorporate the same wording found in the Mapbox Telemetry prompt, but unfortunately there isn’t a way to dynamically set these Info.plist keys based on telemetry settings.

Semantics aside, this is a sound copyediting change.